pkgs/applications/misc/swaynag-battery/default.nix

```
@@ -0,0 +1,22 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "swaynag-battery";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "m00qek";
+    repo = "${pname}";
```

```suggestion
    repo = "swaynag-battery"";
```
substituting every occurrence of pname just makes the code hard to read.